pyexcelerate/Range.py

```diff
@@ -8,6 +11,13 @@
 # to be immutable. Please don't modify attributes after instantiation. :)
 #
 
+# generate list of columns name (COORD2COLUMN[1] => "A") and dict of column name to coord (COLUMN2COORD["A"]=1)
+COORD2COLUMN = list(
+    dict.fromkeys(map("".join, itertools.product([""] + list(string.ascii_uppercase), repeat=3)))
```

Comment:
the product -> dict -> list is to remove the duplicates. The duplicates appears because I take 3 x the product of the list ["", "A", "B", ..., "Z"] which includes a.o. ["", "", "A"] => "A" but also ["","A",""] and ["A","",""] which also resolves to "A" => we want to remove these 2 appearances and it is what is done via the "remove duplicates".

If we do "" + string.ascii_uppercase, we end up with string.ascii_uppercase (as we concatenate the empty string with a string). What we want is a list of all letters from A to Z and the empty string.

is it clearer ?

Reply:
I see, that explanation makes sense, appreciate the comments! Change looks good to me, just one compatibility note.

Reply:
I have also added a test to ensure the properties of COORD2COLUMN  as OK (sorted, unicity, ...)